setup.py

```diff
@@ -350,9 +350,9 @@ def build(self):
         # Fetch all c and cpp files from the subdirs to compile.
         for library_name in self._sub_library_names:
             for extension in ["*.c", "*.cpp"]:
-                self._source_files.extend(glob.glob(
+                self._source_files.extend(sorted(glob.glob(
```

Comment:
> also is consistent order, applied within all versions of Extension() ?

Does the underlying Python code of Extension preserve the order of the source files?